apps/api/src/auth/guest_auth.py

```diff
@@ -32,10 +32,6 @@ class GuestRecord(BaseRecord):
 async def initiate_guest_login(email: EmailStr) -> Optional[str]:
     """Generate a login passphrase to be emailed, save the authentication key,
     and a confirmation token to be saved as a cookie."""
-    if await _get_existing_key(email):
-        # To prevent spammed requests, user must wait until previous key expires
-        return None
-
```

Comment:
Issue: we don't want to regenerate a new key when the user has a recent unexpired key. Instead of changing the return value here, we can just redirect to the verification form from `guest.guest_login` if `confirmation` is `None`